src/components/body/Home/Home.js

```diff
@@ -1,23 +1,44 @@
 import React, {Component} from "react";
-import '../RoutesApp/routes_app.css'
+import '../RoutesApp/RoutesApp.css'
 import file from '../../../assets/file.png';
 import aruba from '../../../assets/aruba_logo.png';
 import sme from '../../../assets/sme_portal.png';
-import Card from '../../ui-modules/Cards/CardUI'
+import Card from '../../modules/HorizontalCard/HorizontalCard'
 import logo from '../../../assets/aruba.png'
 import orange from '../../../assets/orange.png'
 import logo1 from '../../../assets/hpe.jpg'
+import VerticalCard from "../../modules/VerticalCard/VerticalCard";
+import PageTitle from "../../modules/PageTitle/PageTitle";
+
 
 class Home extends Component {
     render() {
-        return (<div className='container-fluid d-flex justify-content-center' >
-            <h2><span className="text-center">HTS </span>Dashboard</h2>
-            <div className='row'>
-                <div className='col-md-4'><Card imgsrc={logo} title='Doc Tool' textContent ='HTS Engineer Portal'/></div>
-                <div className='col-md-4'><Card imgsrc={logo} title='HTS Engineering Portal' textContent ='Admin Portal'/></div>
-                <div className='col-md-4'><Card imgsrc={logo} title='Tableau Dashboards' textContent ='View Dashboards' /></div>
+        return (
+            <div className='container'>
+                <PageTitle title={'HTS Dashboard'} />
+
+                <div className='card'>
+                    <div className='card-body'>
+                        <div className='row'>
+                            <div className='col-sm-4'>
+                                <VerticalCard imgsrc={orange} title='DocTool' toplink='SME Portal' route_toplink = '/doc-tool/sme-portal/home'
+                                                    bottomlink='Engineer Portal' route_bottomlink='/'/>
```

Comment:
whats the route_toplink and route_bottomlink?

Reply:
So a bit of hard coding here.
The Vertical Card is taking 2 child URLs for the sake of space, actually it should take any no. of urls using a loop struct, but I hard coded 2 - toplink and bottomlink . I will ad a TODO here to make it a loop struct in future

Reply:
Nahi samnjha :/